build/bin/release-next.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+echo "Release Type?"
+read -rp "> " rtype
+
+echo grunt version:${rtype}
+grunt version:${rtype}
+version=$(./build/bin/version)
```

Comment:
I'm also a bit weary of changing this script since I know it works as is. I would prefer to just document that you need to run this from the root of the project and we can fix it later if this becomes a problem.